models/issues/issue.go

```diff
@@ -1272,7 +1272,11 @@ func (opts *IssuesOptions) setupSessionNoLimit(sess *xorm.Session) {
 	}
 
 	if len(opts.MilestoneIDs) > 0 {
-		sess.In("issue.milestone_id", opts.MilestoneIDs)
+		if len(opts.MilestoneIDs) == 1 && opts.MilestoneIDs[0] == db.NoneID {
+			sess.And("issue.milestone_id = 0")
```

Comment:
I guess the name "db.NoneID" is not accurate, because the "none" in DB is actually 0.

`db.NoneID` should be something like `RequestNoneID`, because it comes from the user request, and DB layer will use `xxx = 0` for the query.

Reply:
Use `NoConditionID`